Automattic/jetpack
# VideoPress: add tooltips to video settings in Gutenberg

Fixes Automattic/greenhouse#992

#### Changes proposed in this Pull Request:

Add tooltips to the VideoPress block settings in Gutenberg 

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* On a Jetpack site with VideoPress enabled
* Create a new post
* Add a video block and upload or select a video (uploaded with VideoPress)
* Click on the video block
* ✅ Tooltips should appear when hovering text for : 
  * Autoplay
  * Loop
  * Playback Controls
  * Play inline
 * ✅ A "help" icon should appear next to "Preload"
 * ✅ When hovering the preload "help" icon, a tooltip should appear
 * ✅ When selecting "auto" in Preload, a help text should be displayed underneath the combo box

https://user-images.githubusercontent.com/1420594/137141341-9d9f4f7e-0732-405d-9139-87d9b438f22c.mov
